File: 351002/Alexeichik/MyStartRestApplication/discussion/discussion-impl/src/main/java/com/github/Lexya06/startrestapp/discussion/impl/model/entity/realization/Notice.java
package com.github.Lexya06.startrestapp.discussion.impl.model.entity.realization;

import com.github.Lexya06.startrestapp.discussion.api.dto.notice.NoticeState;
import com.github.Lexya06.startrestapp.discussion.impl.model.entity.abstraction.BaseEntity;
import lombok.Getter;
import lombok.Setter;
import org.springframework.data.cassandra.core.mapping.Table;

@Getter
@Setter
@Table("tbl_notice")
public class Notice extends BaseEntity<NoticeKey> {
    private String content;
    private NoticeState state;

}

File: 351003/brezgunov/discussion/src/main/kotlin/com/example/discussion/entity/Notice.kt
package com.example.discussion.entity

import org.springframework.data.cassandra.core.mapping.Column
import org.springframework.data.cassandra.core.mapping.PrimaryKey
import org.springframework.data.cassandra.core.mapping.Table

@Table("tbl_notice")
data class Notice(
    @PrimaryKey
    var id: Long = 0,

    @Column("news_id")
    var newsId: Long = 0,

    @Column("content")
    var content: String = ""
)